superglue/lib/utils/request.ts

```diff
@@ -138,10 +128,10 @@ export function argsForFetch(
   if (method == 'GET' || method == 'HEAD') {
     if (options.body instanceof FormData) {
       const allData = new URLSearchParams(
-        options.body as any
+        options.body as unknown as Record<string, string>
       ).toString()
       // fetchPath will always have atleast /?format=json
-      fetchPath.query = fetchPath.query + '&' + allData
+      fetchPath.set('query', fetchPath.query + '&' + allData)
```

Comment:
`query` as it turns out, is a read only property. I should use the built in `set` method. Same results.